deployment/sklearn/sklearn-census-income-prediction.ipynb

```diff
@@ -0,0 +1,617 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8lCXP3A__nPN"
+      },
+      "source": [
+        "# Census Income Prediction (scikit-learn)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rjbaCPG4_vjE"
+      },
+      "source": [
+        "## 1. Dependencies"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "EFsTgnMe_w9i"
+      },
+      "source": [
+        "This notebook has been tested with **Python 3.8.16** and the following package versions:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "YYZpIDPcky8c"
+      },
+      "outputs": [],
+      "source": [
+        "%%capture\n",
+        "!pip install verta==0.21.1\n",
+        "!pip install wget==3.2"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Eh2rzxWZ_9L0"
+      },
+      "source": [
+        "## 2. Imports"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "PZ9Gbbn-lfxv"
+      },
+      "outputs": [],
+      "source": [
+        "import cloudpickle\n",
+        "import itertools\n",
+        "import os\n",
+        "import pandas as pd\n",
+        "import warnings\n",
+        "import wget\n",
+        "\n",
+        "from sklearn import linear_model\n",
+        "from sklearn.exceptions import ConvergenceWarning\n",
+        "from sklearn.model_selection import train_test_split\n",
+        "from sklearn.preprocessing import LabelEncoder\n",
+        "from verta import Client\n",
+        "from verta.dataset import Path\n",
+        "from verta.environment import Python\n",
+        "from verta.registry import VertaModelBase, verify_io\n",
+        "from verta.utils import ModelAPI\n",
+        "\n",
+        "warnings.filterwarnings('ignore', category=ConvergenceWarning, module='sklearn')\n",
+        "warnings.filterwarnings('ignore', category=FutureWarning, module='sklearn')\n",
+        "warnings.filterwarnings('ignore', category=UserWarning, module='sklearn')"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "GFJlaOwyAJw2"
+      },
+      "source": [
+        "## 3. Verta Set Up"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "TNbSRvSjAMv5"
+      },
+      "outputs": [],
+      "source": [
+        "os.environ['VERTA_HOST'] = ''\n",
+        "os.environ['VERTA_EMAIL'] = ''\n",
+        "os.environ['VERTA_DEV_KEY'] = ''"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Twn7sVWrAn4e"
+      },
+      "outputs": [],
+      "source": [
+        "client = Client(os.environ['VERTA_HOST'])\n",
```

Comment:
Can be simplified to just `client = Client()`

Reply:
Didn't know. Done :+1: